src/test/java/com/microsoft/azure/servicebus/jms/aad/Utility.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoft.azure.servicebus.jms.aad;
+
+import com.microsoft.azure.servicebus.jms.ServiceBusJmsConnectionFactorySettings;
+
+public class Utility {
+	
+	public static final ServiceBusJmsConnectionFactorySettings CONNECTION_SETTINGS;
+	public static final String TENANT_ID;
+	public static final String CLIENT_SECRET;
+	public static final String CLIENT_ID;
+	public static final String HOST;
+	public static final long DEFAULT_TIMEOUT;
+	
+    static {
+        
+        CONNECTION_SETTINGS = new ServiceBusJmsConnectionFactorySettings(120000, true);
```

Comment:
>(120000, true); [](http://example.com/codeflow?start=72&length=15)

nit: can you qualify what these two arguments represent?

Reply:
what I meant here was in c#, there is a way to specify the variable names you are passing these values for:
for ex in c#, you could write the above as new ServiceBusJmsConnectionFactorySettings(**connectionIdleTimeoutMS**: 120000, **traceFrames**: true);

Reply:
I was wondering if we could qualify it similarly here in Java as well. It helps to know what are the arguments being passed for quickly.

Reply:
I try seeing if that was possible so far I cannot find it by it self but in intellij ide if you have enable hint you can see it like that format.